shared/src/adapters/neo4j/queries/workflowRuns/addEvent.mapper.ts

```
@@ -0,0 +1,153 @@
+        type: "tool.result",
+        content: neo4jEvent.content,
+        metadata: {
+          toolCallId: neo4jEvent.toolCallId,
+          toolName: neo4jEvent.toolName,
+        },
+      }
+
+    default:
+      // TypeScript will ensure this is exhaustive
+      const _exhaustive: never = neo4jEvent
+      throw new Error(`Unknown event type: ${(_exhaustive as AnyEvent).type}`)
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)